apps/openmw/mwworld/projectilemanager.cpp

```diff
@@ -406,29 +406,27 @@ namespace MWWorld
             // TODO: use a proper btRigidBody / btGhostObject?
             MWPhysics::PhysicsSystem::RayResult result = mPhysics->castRay(pos, newPos, caster, 0xff, MWPhysics::CollisionType_Projectile);
 
+            bool hitDeadActor = !result.mHitObject.isEmpty() && result.mHitObject.getClass().isActor() && result.mHitObject.getClass().getCreatureStats(result.mHitObject).isDead();
```

Comment:
From a quick test everything seems to be working, so you can delete this change.